src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -130,6 +132,10 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
             throw new \Exception('Function not found', 404);
         }
 
+        if ($isResourceBlocked($project, RESOURCE_TYPE_FUNCTIONS, $functionId)) {
+            throw new \Exception('Function blocked', 403);
```

Comment:
```suggestion
            throw new \Exception('Function blocked');
```
Since this is a worker, the exception shouldn't use a HTTP error code. The exception on L132 is also incorrect.